src/popup/components/AppSettings.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="content">
+    <div class="p-3 container">
+      <p class="font-weight-bold align-self-center">App Settings</p>
+
+      <table class="settings-list">
+        <tr>
+          <td>
+            <toggle-button
+              :value="showSLButton"
+              :sync="true"
+              color="#b02a8f"
+              :width="30"
+              :height="18"
+              @change="handleToggleSLButton()"
+            />
+          </td>
+          <td>
+            Show SimpleLogin button on email input fields<br />
+            <small>
+              SimpleLogin button allows you to quickly create random alias
+              without having to navigating to SimpleLogin application.
+              <a :href="SLButtonReport" v-show="showSLButton" target="_blank">
+                <br/><font-awesome-icon icon="bug" /> Report a problem with this feature
+              </a>
+            </small>
+          </td>
+        </tr>
+
+        <tr v-show="showSLButton">
+          <td>
+            <toggle-button
+              :value="SLButtonPosition === 'right-outside'"
+              :sync="true"
+              color="#b02a8f"
+              :width="30"
+              :height="18"
+              @change="handleToggleSLButtonOutside()"
+            />
+          </td>
+          <td>
+            Place SimpleLogin button outside the input<br />
+            <small
```

Comment:
can we change the text here to "Display the SimpleLogin button next to the email field instead of inside the field. This can avoid having overlapping buttons with other extensions like Dashlane, LastPass, etc"?